assets/js/util/date-range/get-available-date-ranges.js

```diff
@@ -1,5 +1,5 @@
 /**
- * Utility functions.
+ * `getAvailableDateRanges` utility.
```

Comment:
the diff here looks odd because the new `index.js` only has the same File Header, and the methods inside it have now been extracted out (such as this one, which the diff is picking up as the new file)